pkg/pillar/cmd/zedagent/handlenetworkinstance.go

```diff
@@ -829,20 +829,20 @@ func protoEncodeAppFlowMonitorProto(ipflow types.IPFlow) *flowlog.FlowMessage {
 
 func sendFlowProtobuf(protoflows *flowlog.FlowMessage) {
 
-	flowQ.PushBack(*protoflows)
+	flowQ.PushBack(protoflows)
 
 	for flowQ.Len() > 0 {
 		ent := flowQ.Front()
-		pflows := ent.Value.(flowlog.FlowMessage)
+		pflowsPtr := ent.Value.(*flowlog.FlowMessage)
```

Comment:
This will presumably assert because you are casting to a different type, yet there is no change in this PR when ent.Value is assigned.

Reply:
When we push into Q - we are pushing the Ptr
ent.Value is an interface{} .. So we need to cast it back as a Ptr

Reply:
Ah got it